### PR TITLE
fix: courriers indexation

### PR DESCRIPTION
**Pull request by revolunet**
- cleanup du fichier courriers.json
 - remove need of manual "build"
 - html is directly extracted at indexation time

same coming for fiches sp

### CODE REVIEW

**Review by lionelB — COMMENTED** [review]
should we add it to .gitignore

**Comment by revolunet** [comment]
the file is removed forever so no need to ignore it ?

**Comment by lionelB** [comment]
each time someone launch `yarn build` it will generate it.

**Comment by revolunet** [comment]
non en fait là ce fichier intermediaire n'est plus généré. l'extract HTML est réalisé au moment de l'indexation

File: packages/code-du-travail-data/indexing/populate.js [hunk 1]
```diff
@@ -136,30 +137,30 @@ function* cdtnDocumentsGen() {
...
-      description,
-      text: questions.join("\n"),
-      html,
-      filename,
-      date: date_redaction,
-      editor: source,
-      author: redacteur,
-      tags: flattenTags(tags)
-    })
+  yield getCourriers().then(courriers =>
```
Comment (lionelB) [inline packages/code-du-travail-data/indexing/populate.js]: yielding a promise unwrap it ? 
o_O  didn't know we could do that :)

Comment (revolunet) [reply packages/code-du-travail-data/indexing/populate.js]: no you cant :)

**Comment by revolunet** [comment]
up :)

File: packages/code-du-travail-data/indexing/slug_checker.js [hunk 2]
```diff
@@ -1,15 +1,19 @@
 import { cdtnDocumentsGen, getDuplicateSlugs } from "./populate";
 import { logger } from "./logger";
 
-const documents = cdtnDocumentsGen();
-const duplicateSlugs = getDuplicateSlugs(documents);
+const checkDuplicates = async () => {
+  const documents = await cdtnDocumentsGen();
+  const duplicateSlugs = getDuplicateSlugs(documents);
```
Comment (lionelB) [inline packages/code-du-travail-data/indexing/slug_checker.js:6]: Does `await` missing here ?

**Review by lionelB — APPROVED** [review]

**Comment by revolunet** [comment]
thank a ton !

**Pull request merged**